### PR TITLE
Add about twenty new items to FAQ

### PR DESCRIPTION
**Pull request by lilin90**
Via https://github.com/pingcap/docs-cn/pull/501 and https://github.com/pingcap/docs-cn/pull/507. PTAL

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -107,6 +107,37 @@ Currently, TiDB does not support `select into outfile`. You can use the followin
...
 
+#### What is the TiDB version management strategy for production environment? How to avoid frequent upgrade?
+
+Currently, TiDB has a standard management of various versions. Each release contains a detailed change log and [release notes](https://github.com/pingcap/TiDB/releases). Whether it is necessary to upgrade in the production environment depends on the application system. It is recommended to learn the details about the functional differences between the previous and later versions before upgrading.
+
+Take `Release Version: v1.0.3-1-ga80e796` as an example of version number description:
+
+- `v1.0.3` represents the standard GA version.
+- `-1` represents the current version has one commit.
```
Comment (CaitinChen) [inline FAQ.md]: replace "represents" with "indicates"

File: FAQ.md [hunk 2]
```diff
@@ -107,6 +107,37 @@ Currently, TiDB does not support `select into outfile`. You can use the followin
...
+The TiDB community is highly active. After the GA release, the engineers have been keeping optimizing and fixing bugs. Therefore, the TiDB version is updated quite fast. If you want to keep informed of the latest version, see [TiDB Weekly update](https://pingcap.com/weekly/).
+
+It is recommended to deploy the TiDB cluster using the latest version of TiDB-Ansible, which will also be updated along with the TiDB version. Besides, TiDB has a unified management of the version number after GA release. You can view the version number using the following two methods:
+
+- `select tidb_version()`
+- `tidb-server -V`
+
+#### What's the recommended solution for the deployment of three geo-distributed data centers?
+
+The architecture of TiDB guarantees that it fully supports geo-distributed and multi-activeness. Your data and applications are always-on. All the outages are transparent to your applications and your data can recover automatically. The operation depends on the network latency and stability. It is recommended to keep the latency within 5ms. Currently, we already have similar use cases. For for details, contact info@pingcap.com.
```
Comment (CaitinChen) [inline FAQ.md]: Replace "geo-distributed" with "geo-distribution"

Comment (CaitinChen) [inline FAQ.md]: In the last sentence, delete "for".

File: FAQ.md [hunk 3]
```diff
@@ -144,6 +175,18 @@ Yes. Currently, you can only update the global number of replicas. When started
 
 Make sure that the total number of TiKV instances is always greater than or equal to the number of replicas you set. For example, 3 replicas need 3 TiKV instances at least. Additional storage requirements need to be estimated before increasing the number of replicas. For more information about pd-ctl, see [PD Control User Guide](tools/pd-control.md).
 
+#### How to delete the monitoring data of a cluster node that is offline?
+
+The offline node usually refers to the TiKV node. You can determine whether the offline process is finished by the pd-ctl or the monitor. After the node is offline, perform the following steps:
```
Comment (CaitinChen) [inline FAQ.md]: The offline node "is usually referred to as" the TiKV node.

**Comment by dbaoutdo** [comment]
LGTM

**Comment by CaitinChen** [comment]
LGTM

**Pull request merged**